src/azure-cli/azure/cli/command_modules/vm/generated/_help.py

```diff
@@ -0,0 +1,65 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['vm ssh-public-key'] = """
+    type: group
+    short-summary: Manage ssh public key with vm
+"""
+
+helps['vm ssh-public-key list'] = """
+    type: command
+    short-summary: "Lists all of the SSH public keys in the specified resource group. Use the nextLink property in the \
+response to get the next page of SSH public keys. And Lists all of the SSH public keys in the subscription. Use the \
+nextLink property in the response to get the next page of SSH public keys."
+"""
+
+helps['vm ssh-public-key show'] = """
+    type: command
+    short-summary: "Retrieves information about an SSH public key."
+    examples:
+      - name: Get an ssh public key.
+        text: |-
+               az vm ssh-public-key show --resource-group "myResourceGroup" --name "mySshPublicKeyName"
+"""
+
+helps['vm ssh-public-key create'] = """
+    type: command
+    short-summary: "Creates a new SSH public key resource."
```

Comment:
this will be updated right? to support uploading public key except generation

Reply:
all help is in manual folder now